manim/utils/space_ops.py

```diff
@@ -775,3 +778,67 @@ def perpendicular_bisector(
     direction = np.cross(p1 - p2, norm_vector)
     m = midpoint(p1, p2)
     return [m + direction, m - direction]
+
+
+def projected_vector(vec1: np.ndarray, vec2: np.ndarray):
+    """Returns a vector that is projected onto another vector.
+
+    Parameters
+    ----------
+    vec1
+        The vector to be projected onto.
+    vec2
+        The vector being projected.
+
+    Returns
+    -------
+    numpy array
+        The component vector of the second vector
+        in the direction of the first vector
+    """
+    dot_prod = np.dot(vec1, vec2)
+    length_prod = np.linalg.norm(vec1) ** 2
+    return vec1 * dot_prod / length_prod
+
+
+def cartesian_to_spherical(point: Union[list, np.ndarray]):
```

Comment:
the return type is an `np.ndarray` here
```suggestion
def cartesian_to_spherical(point: Sequence[float]):
```